lib/tpa/transmogrifiers/replace_2q_repositories.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
+
+
+from ..transmogrifier import Transmogrifier, opt
+from ..changedescription import ChangeDescription
+from ..checkresult import CheckResult
+
+
+class Replace2qRepositories(Transmogrifier):
+    def __init__(self):
+        # this dict matches the one for new clusters in lib/tpaexec/architectures.py
+        self.postgres_repos = {
+            "postgresql": ["standard"],
+            "edbpge": ["standard"],
+            "pgextended": ["standard", "postgres_extended"],
+            "epas": ["enterprise"],
+        }
+
+    @classmethod
+    def options(cls):
+        return {
+            **opt(
+                "--replace-2q-repositories",
+                action="store_true",
+                help="Replace 2q repositories in config.yml with EDB repositories",
+            )
+        }
+
+    def is_applicable(self, cluster):
+        return True
```

Comment:
This method will return always True, is this a left over?

Reply:
this method should return true if change would be applied to the cluster.
in this case maybe we could check that 2q is not empty and that arch is not pgd-always-on

Reply:
Thanks for catching this; I'd moved the `return True` that should have been at the end to the beginning for debugging. There is actually some careful logic in is_applicable, it was just being bypassed.